testfiles-gd/support/pgfgd-regression-test.tex

```diff
@@ -1,18 +1,5 @@
 \input regression-test.tex
 
-\catcode`\@=11 % \makeatletter
-
-% this macro should be used _after_ graphdrawing library is loaded
-\def\pgfgdBeforeBeginDocument{
-  % redirect error
-  \def\pgfutil@packageerror#1#2#3{\immediate\write17{Package #1 Error: #2.}}
```

Comment:
@hmenke It's the removal of this line that broken the test. The aim was to test that specific usage does throw errors.